lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -27,7 +28,7 @@ has _ua  => sub {
 
 sub add {
   my ($self, $moniker, @files) = @_;
-
+  @files = $self->_check_for_wildcards(\@files) if(grep(/\*\./, @files));
```

Comment:
You don't need to check for wildcards here _and_ inside _check_for_wildcards(). Just do it inside _check_for_wildcards(). I would also just pass inn @files and not the ref. If you want to pass in \@files. Or maybe do this instead:

    @files = map { $self->_glob($_) } @files;